### DISCUSSION TITLE
Creating Custom Style Manager

### DISCUSSION THREAD

**Discussion by RakulAgn** [GrapesJS/grapesjs#5128] [General]
Is the Possible to Create Custom Style Manager Without any Sectors 
Expecting Like This Below
![image](https://github.com/GrapesJS/grapesjs/assets/119593989/14d68e56-3bb1-4e47-9b07-4437ab4d47fc)

**Comment by artf** [comment]
You can create your [own Style Manager](https://grapesjs.com/docs/modules/Style-manager.html#custom-style-manager) and just ignore rendering sectors
